index.html

```diff
@@ -20,26 +16,227 @@
     <!-- Custom styles for this template -->
     <link href="css/styles.css" rel="stylesheet">
 
-
     <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
     <!--[if lt IE 9]>
       <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-    <![endif]-->
+    <![endif]-->    
 
 </head>
 
-  <body>
+    <body class="lista">
```

Comment:
Your code looks very clean and easy to read, thanks! Using indentations really makes a huge difference in the code readability. :D